src/components/Documentation/Layout/SidebarMenu/index.tsx

```diff
@@ -92,6 +112,14 @@ const SidebarMenuItem: React.FC<ISidebarMenuItemProps> = ({
         className={className}
         onClick={currentLevelOnClick}
       >
+        <span
+          className={styles.absBeforeIconOverlay}
+          onClick={bulletIconClick}
+          onKeyDown={bulletIconClick}
+          data-path={path}
+          role="button"
+          tabIndex={0}
+        ></span>
```

Comment:
I think it would be preferable not to add an overlay element when we have full control over the arrow already.  
Maybe we could apply the styles currently on the `::before` to this element?

Reply:
@rogermparent Can you suggest any work around to detect the click for the arrow?